src/main/java/edu/ksu/canvas/attendance/model/AttendanceSummaryModel.java

```diff
@@ -18,47 +18,56 @@
         private final long courseId;
         private final long sectionId;
         private final long studentId;
+        private final String SisUserId;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "SisUserId" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00116)